jax/_src/lib/xla_bridge.py

```diff
@@ -212,20 +220,29 @@ def register_backend_factory(name, factory, *, priority=0):
                          priority=100)
 
 
-def make_gpu_client(platform_name=None):
+def make_gpu_client(*, platform_name, visible_devices_flag):
+  visible_devices = getattr(FLAGS, visible_devices_flag, "all")
+  # Pass allowed_devices unconditionally when jaxlib 0.3.15 is the minimum.
+  kwargs = {}
+  if visible_devices != "all":
+    kwargs["allowed_devices"] = {int(x) for x in visible_devices.split(",")}
```

Comment:
Instead of just `.split()`, if someone does `"1, 2, 3"`, then you might have to strip the whitespaces.

So probably add `.strip` too?

Reply:
probably okay since `int(x)` would ignore whitespaces?

Reply:
Oh wow, that works!